src/gui/creds/webflowcredentials.cpp

```diff
@@ -162,8 +162,8 @@ void WebFlowCredentials::askFromUser() {
             _askDialog->setUrl(url);
         }
 
-        QString msg = tr("You have been logged out of %1 as user %2. Please login again.")
-                          .arg(_account->displayName(), _user);
+        QString msg = tr("You have been logged out of your account %1 at %2. Please login again.")
```

Comment:
at --> as?

Reply:
`%2` refers to the server and not the user itself, so the string would be "You have logged out of your account NimishaAccount at cloud.nextcloud.com"